ui/components/component-library/avatar-icon/avatar-icon.stories.js

```diff
@@ -107,80 +107,82 @@ export const SizeStory = (args) => (
 SizeStory.storyName = 'Size';
 
 SizeStory.args = {
-  iconName: ICON_NAMES.CONFIRMATION,
+  iconName: IconName.Confirmation,
 };
 
-export const IconName = (args) => (
+export const IconNameStory = (args) => (
   <Box display={DISPLAY.FLEX} gap={1}>
     <AvatarIcon
       color={IconColor.primaryDefault}
       backgroundColor={BackgroundColor.primaryMuted}
-      iconName={ICON_NAMES.SWAP_HORIZONTAL}
+      iconName={IconName.SwapHorizontal}
       {...args}
     />
     <AvatarIcon
       color={IconColor.successDefault}
       backgroundColor={BackgroundColor.successMuted}
-      iconName={ICON_NAMES.CONFIRMATION}
+      iconName={IconName.Confirmation}
       {...args}
     />
     <AvatarIcon
       color={IconColor.infoDefault}
       backgroundColor={BackgroundColor.infoMuted}
-      iconName={ICON_NAMES.INFO}
+      iconName={IconName.Info}
       {...args}
     />
     <AvatarIcon
       color={IconColor.warningDefault}
       backgroundColor={BackgroundColor.warningMuted}
-      iconName={ICON_NAMES.WARNING}
+      iconName={IconName.Warning}
       {...args}
     />
     <AvatarIcon
       color={IconColor.errorDefault}
       backgroundColor={BackgroundColor.errorMuted}
-      iconName={ICON_NAMES.DANGER}
+      iconName={IconName.Danger}
       {...args}
     />
   </Box>
 );
 
+IconNameStory.storyName = 'Icon Name';
+
 export const ColorAndBackgroundColor = (args) => (
   <Box display={DISPLAY.FLEX} gap={1}>
     <AvatarIcon
       color={IconColor.primaryDefault}
       backgroundColor={BackgroundColor.primaryMuted}
-      iconName={ICON_NAMES.SWAP_HORIZONTAL}
+      iconName={IconName.SwapHorizontal}
       {...args}
     />
     <AvatarIcon
       color={IconColor.primaryInverse}
       backgroundColor={BackgroundColor.primaryDefault}
-      iconName={ICON_NAMES.SWAP_HORIZONTAL}
+      iconName={IconName.SwapHorizontal}
       {...args}
     />
     <AvatarIcon
       color={IconColor.successDefault}
       backgroundColor={BackgroundColor.successMuted}
-      iconName={ICON_NAMES.CONFIRMATION}
+      iconName={IconName.Confirmation}
       {...args}
     />
     <AvatarIcon
       color={IconColor.infoDefault}
       backgroundColor={BackgroundColor.infoMuted}
-      iconName={ICON_NAMES.INFO}
+      iconName={IconName.Info}
       {...args}
     />
     <AvatarIcon
       color={IconColor.warningDefault}
       backgroundColor={BackgroundColor.warningMuted}
-      iconName={ICON_NAMES.WARNING}
+      iconName={IconName.Warning}
       {...args}
     />
     <AvatarIcon
       color={IconColor.errorDefault}
       backgroundColor={BackgroundColor.errorMuted}
-      iconName={ICON_NAMES.DANGER}
+      iconName={IconName.Danger}
       {...args}
     />
   </Box>
```

Comment:
## After
Icon Name story still works and shows up in docs

https://user-images.githubusercontent.com/8112138/227677785-3782150b-c1db-4b98-8895-e3d254bb72ed.mov